app/lib/search/search_form.dart

```diff
@@ -188,14 +190,18 @@ class SearchForm {
     return values.isEmpty ? null : values.first;
   }
 
+  /// Returns whether the search is on Flutter Favorites.
+  bool get isFlutterFavorite =>
+      tagsPredicate.isRequiredTag(PackageTags.isFlutterFavorite);
```

Comment:
This doesn't affect when users type-in `is:flutter-favorite` in the search query, right?

`TagPredicate` is more of a `SearchContext` really, right? It's the context within which search is presented, is it not?

Just trying to understand, not saying we have to refactor now, just saying wow... there is a lot of implicit knowledge here that is hard to follow (was also hard to follow when I did FF listing and made it worse).

Reply:
> This doesn't affect when users type-in is:flutter-favorite in the search query, right?

Correct.

> TagPredicate is more of a SearchContext really, right? It's the context within which search is presented, is it not?

Yes, we should probably move this toward the `Place` / `Page` concept that I've talk about recently.